packages/guides/src/Twig/AssetsExtension.php

```diff
@@ -157,6 +160,10 @@ public function renderMenu(array $context, string $menuType, int $maxMenuCount =
     /** @param array{env: RenderContext} $context */
     public function renderLink(array $context, string $url, string|null $anchor = null): string
     {
+        if (filter_var($url, FILTER_VALIDATE_URL) !== false) {
```

Comment:
Should this not be part of the urlGenerator logic?

Reply:
True, moved it there